dts/arm/microchip/sam/sama7g5.dtsi

```diff
@@ -446,6 +446,90 @@
 			#interrupt-cells = <4>;
 		};
 
+		mcan0: mcan@e0828000 {
+			compatible = "atmel,sam-can";
+			reg = <0xe0828000 0x100>, <0x100000 0x7800>, <0xe1626030 0x4>;
+			reg-names = "m_can", "message_ram", "sram_sel";
+			interrupt-parent = <&gic>;
+			interrupts = <GIC_SPI 61 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>,
+				     <GIC_SPI 123 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
+			interrupt-names = "int0", "int1";
+			clocks = <&pmc PMC_TYPE_GCK 61>;
+			clock-names = "gck";
+			bosch,mram-cfg = <0x3400 4 4 64 4 4 32 32>;
+			status = "disabled";
+		};
+
+		mcan1: mcan@e082c000 {
+			compatible = "atmel,sam-can";
+			reg = <0xe082c000 0x100>, <0x100000 0xbc00>, <0xe1626030 0x4>;
+			reg-names = "m_can", "message_ram", "sram_sel";
+			interrupt-parent = <&gic>;
+			interrupts = <GIC_SPI 62 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>,
+				     <GIC_SPI 124 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
+			interrupt-names = "int0", "int1";
+			clocks = <&pmc PMC_TYPE_GCK 62>;
+			clock-names = "gck";
+			bosch,mram-cfg = <0x7800 0 0 64 0 0 32 32>;
```

Comment:
The configuration of mcan1 to mcan5 look a bit odd to me as no RX filters are available? Also, the mcan1 instance only has 4 x 11-bit + 4 x 29-bit filters, but 64 RX FIFO elements and 32 TX FIFO elements, which seems like an odd choice to me?